pom.xml

```diff
@@ -317,6 +340,129 @@ limitations under the License.
+                      <value>${accumulo.minicluster.instance.name}</value>
+                    </property>
+                    <property>
+                      <name>accumulo.minicluster.root.password</name>
+                      <value>${accumulo.minicluster.root.password}</value>
+                    </property>
+                    <property>
+                      <name>accumulo.minicluster.zookeeper.port</name>
+                      <value>${accumulo.minicluster.zookeeper.port}</value>
+                    </property>
+                  </properties>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+          <plugin>
+            <groupId>org.apache.accumulo</groupId>
+            <artifactId>accumulo-maven-plugin</artifactId>
+            <version>${accumulo.version}</version>
+            <configuration>
+              <instanceName>${accumulo.minicluster.instance.name}</instanceName>
+              <rootPassword>${accumulo.minicluster.root.password}</rootPassword>
+              <zooKeeperPort>${accumulo.minicluster.zookeeper.port}</zooKeeperPort>
+            </configuration>
+            <executions>
+              <execution>
+                <id>start-plugin</id>
+                <goals>
+                  <goal>start</goal>
+                </goals>
+                <phase>pre-integration-test</phase>
+              </execution>
+              <execution>
+                <id>stop-plugin</id>
+                <goals>
+                  <goal>stop</goal>
+                </goals>
+                <phase>post-integration-test</phase>
+              </execution>
+            </executions>
+          </plugin>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-failsafe-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>integration-test</id>
+                <goals>
+                  <goal>integration-test</goal>
+                  <goal>verify</goal>
+                </goals>
+                <configuration>
+                  <systemPropertyVariables>
+                    <accumulo.minicluster.instance.name>${accumulo.minicluster.instance.name}</accumulo.minicluster.instance.name>
+                    <accumulo.minicluster.root.password>${$accumulo.minicluster.root.password}</accumulo.minicluster.root.password>
+                    <accumulo.minicluster.zookeeper.port>${accumulo.minicluster.zookeeper.port}</accumulo.minicluster.zookeeper.port>
+                  </systemPropertyVariables>
+                  <includes>
+                    <include>**/*IT.java</include>
+                  </includes>
+                  <skipTests>${skipTests}</skipTests>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+          <plugin>
+            <groupId>com.google.code.sortpom</groupId>
```

Comment:
Why do we have sort pom in the profile. Isn't this already (or should be) done outside of a profile?